Backend/app/rider-platform/rider-app-drainer/src/DBSync/Create.hs

```diff
@@ -76,6 +76,7 @@ runCreateCommands cmds streamKey = do
     |::| runCreateInKafkaAndDb dbConf streamKey ("Location" :: Text) [(obj, val, entryId, LocationObject obj) | (CreateDBCommand entryId _ _ _ _ (LocationObject obj), val) <- cmds]
     |::| runCreateInKafkaAndDb dbConf streamKey ("LocationMapping" :: Text) [(obj, val, entryId, LocationMappingObject obj) | (CreateDBCommand entryId _ _ _ _ (LocationMappingObject obj), val) <- cmds]
     |::| runCreateInKafkaAndDb dbConf streamKey ("NextBillionData" :: Text) [(obj, val, entryId, NextBillionDataObject obj) | (CreateDBCommand entryId _ _ _ _ (NextBillionDataObject obj), val) <- cmds]
+    |::| runCreateInKafkaAndDb dbConf streamKey ("AutoCompleteData" :: Text) [(obj, val, entryId, NextBillionDataObject obj) | (CreateDBCommand entryId _ _ _ _ (NextBillionDataObject obj), val) <- cmds]
```

Comment:
I think here **NextBillionDataObject** should be **AutoCompleteDataObject**

Reply:
Done